docs/getting-started.md

```diff
@@ -24,40 +24,47 @@ You can optionally persist the output of the `onit completion` command to your s
 
 ## Usage
 
+### Development Cluster
+
+To setup a cluster for development, after having pushed all the new images to `kind` run:
+```bash
+onit create cluster --set onos-cli.enabled=true
+```
+
 ### Integration Tests
 
 To run a suite of tests, use the `onit run test` command, providing a test image to run:
 
 ```bash
-onit test --image onosproject/onos-tests:latest
+onit test --image <test-image>
+```
+
+For example, to run onos-config suite tests using [kind] cluster:
```

Comment:
this could be done by just letting it pull automatically

Reply:
Will update it accordingly